lib/transmissions.js

```diff
@@ -70,9 +73,16 @@ module.exports = function(client) {
         return Promise.reject(new Error('transmission object is required')).asCallback(callback);
       }
 
+      // format the payload if we are given an array of recipients
+      if (_.has(modifiedTransmission, 'recipients') &&
+          _.isArray(modifiedTransmission.recipients) &&
+          !_.has(modifiedTransmission.recipients[0], 'list_id')) {
+        formatPayload(modifiedTransmission);
+      }
+
       const reqOpts = {
         uri: api,
-        json: transmission,
+        json: modifiedTransmission,
```

Comment:
I think it'd be nice to move all of the checks into `formatPayload`, and also collapse the first `_.has` check and the `_.isArray` check because `_.isArray` will return false if `transmission.recipients` isn't defined.

Then you don't need the const here, you can do the clone in formatPayload, and do 2 individual checks with early returns, and comments about why for each (e.g. "  // return as-is if recipients is not an array") which I think will be a little clearer.